website/docs/r/registry_module.html.markdown

```diff
@@ -125,6 +155,9 @@ The following arguments are supported:
 * `namespace` - (Optional) The namespace of a public registry module. It can be used if `module_provider` is set and `registry_name` is public.
 * `registry_name` - (Optional) Whether the registry module is private or public. It can be used if `module_provider` is set.
 
+The `test_config` block supports
+* `tests_enabled` - (Required) Specifies whether tests run for the registry module. Tests are only supported for branch-based publishing.
```

Comment:
the implementation sets both `test_config` and `tests_enabled` as optional attributes, but here it is state as required...update to reflect same requirement.

Reply:
Updated